packages/appsync-emulator-serverless/__test__/schemaTest.test.js

```diff
@@ -3,7 +3,17 @@ const { graphql } = require('graphql');
 const { subscribe } = require('graphql/subscription');
 const gql = require('graphql-tag');
 const { decoded: jwt } = require('../testJWT');
-const dynamodbEmulator = require('@conduitvc/dynamodb-emulator/client');
+const nock = require('nock');
+// const dynamodbEmulator = require('@conduitvc/dynamodb-emulator/client');
```

Comment:
@pianomansam -- this is why the tests are failing

Reply:
🤦‍♂️

Reply:
I approved it 😄

Reply:
I undid that change but it's still not passing the tests 😢